Readding the Provision section to the workshop (#1693)
aws/eks-anywhere: docs/content/en/docs/workshops/provision/vSphere-prepeartion.md

> ---

Reviewer: Could you change the filename from "vSphere-prepeartion.md" to  "vSphere-preparation.md"?

> title: "Preparing Needed for hosting EKS-A on vSphere"

Reviewer: ```suggestion
title: "Preparation needed for hosting EKS Anywhere on vSphere"
```

> title: "Preparing Needed for hosting EKS-A on vSphere"

Reviewer 2: ```suggestion
title: "Preparing needed for hosting EKS Anywhere on vSphere"
```

> linkTitle: "vSphere-prep"

Reviewer 2: I'm not sure why this would be hyphenated
```suggestion
linkTitle: "vSphere prep"
```
